index.js

```diff
@@ -1,6 +1,7 @@
 var onIdle = require('on-idle')
 var assert = require('assert')
 
+var disabled = window.localStorage.DISABLE_NANOTIMING === true
```

Comment:
this depends on `window.localStorage` existing - does it need `window && window.localStorage && ...` ?